src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -50,6 +54,9 @@
     protected FilteredList<Team> filteredTeamList;
     protected FilteredList<Mentor> filteredMentorList;
 
+    protected SortedList<Team> sortedTeam;
+    protected SortedList<Team> topKTeams;
```

Comment:
Should this go under the teamList class? Seems like it is very independent of the other ModelManager code.